web/themes/custom/recipe_theme/src/sass/content/_node.landing-page.scss

```diff
@@ -3,3 +3,9 @@
   margin-bottom: 3rem;
   opacity: 0.5;
 }
+
+.landing-page__body {
+  text-align: center;
+  margin-left: 6rem;
```

Comment:
Have you validated if this looks properly on mobile devices?

Reply:
I've added a media query to improve the appearance on smaller screens.